### DISCUSSION TITLE
Impossible to use prisma.$queryRaw with template literals and variables?!

### DISCUSSION THREAD

**Discussion by marcusdiy** [prisma/prisma#21122] [Q&A]
### Bug description

Hello, I'm trying to export my database data. 
For that i would like to use $queryRaw to select rows but it fails and I cant make it work in any way
Its simple if done using sql, but prisma seems to make it difficult!

### How to reproduce

Use any prisma project, and replace Identity with any table name. 
First query will work, the next one. No.

```
(async () => {
  const tableData = await prisma.$queryRaw`SELECT * FROM \`Identity\``;
  console.log('Found rows', tableData.length);

  const tableName = 'Identity';
  const tableData2 = await prisma.$queryRaw`SELECT * FROM \`${tableName}\``;
  console.log('Found rows', tableData2.length);
})();
```

### Prisma information

--

### Environment & setup

--

### Prisma Version

5.2.0

**Comment by drewcorlin1** [comment]
Have you tried using the [tagged template helpers](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access#tagged-template-helpers)?


Something like:
```typescript
const tableData2 = await prisma.$queryRaw`SELECT * FROM \`${Prisma.sql`${tableName}`}\``;
```

**Comment by ludralph** [comment]
Hi @marcusdiy 👋 

Thank you for raising this question.

Please look at this [solution](https://github.com/prisma/prisma/issues/9765#issuecomment-1528729000) for working with dynamic table names.

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
